yarn-project/acir-simulator/package.json

```diff
@@ -35,7 +35,7 @@
     "@aztec/circuits.js": "workspace:^",
     "@aztec/foundation": "workspace:^",
     "@aztec/types": "workspace:^",
-    "acvm_js": "github:noir-lang/acvm-simulator-wasm#b9d9ca9dfc5140839f23998d9466307215607c42",
+    "acvm_js": "github:dbanks12/acvm-simulator-wasm#db/init-sim-backend",
```

Comment:
It feels a bit bad that we can't get this in quickly even with agreement from the component owner, but I get the practicality